gdal-sys/src/ogr.rs

```diff
@@ -42,10 +49,17 @@ extern {
     pub fn OGR_G_DestroyGeometry(hGeom: *mut c_void);
     pub fn OGR_Fld_GetNameRef(hDefn: *const c_void) -> *const c_char;
     pub fn OGR_Fld_GetType(hDefn: *const c_void) -> c_int;
+    pub fn OGR_Fld_Create(pszName: *const c_char, eType: c_int) -> *const c_void;
+    pub fn OGR_Fld_GetWidth(hDefn: *const c_void) -> c_int;
+    pub fn OGR_Fld_GetPrecision(hDefn: *const c_void) -> c_int;
+    pub fn OGR_Fld_SetWidth(hDefn: *const c_void, nNewWidth: c_int) -> c_void;
+    pub fn OGR_Fld_SetPrecision(hDefn: *const c_void, nNewPrecision: c_int) -> c_void;
+    pub fn OGR_Fld_Destroy(hDefn: *mut c_void) -> c_void;
     pub fn OGRFree(ptr: *mut c_void);
     pub fn VSIFree(ptr: *mut c_void);
 }
 
+pub const OFT_INTEGER:            c_int = 0;
```

Comment:
I was also thinking about moving OFT types into an enum, so no problem, I will do it.